.github/workflows/ci.yml

```diff
@@ -0,0 +1,18 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - main
+      - 2201.[0-9]+.x
+  repository_dispatch:
+    types: check_connector_for_breaking_changes
+
+jobs:
+  call_workflow:
+    name: Run Connector Build Workflow
+    if: ${{ github.repository_owner == 'ballerina-platform' }}
+    uses: ballerina-platform/ballerina-library/.github/workflows/build-connector-template.yml@main
```

Comment:
@Nuvindu We should be using stdlib templetes (instead of using connector templates) in our workflows

Reply:
Yeah, will fix that.